web/sharings/sharings.go

```diff
@@ -13,31 +13,27 @@ import (
 
 // SharingAnswer handles a sharing answer from the sharer side
 func SharingAnswer(c echo.Context) error {
-
 	var err error
+	var u string
 
-	state := c.FormValue("state")
-	clientID := c.FormValue("client_id")
-	scope := c.FormValue("scope")
-	accessCode := c.FormValue("access_code")
+	state := c.QueryParam("state")
+	clientID := c.QueryParam("client_id")
+	accessCode := c.QueryParam("access_code")
 
 	instance := middlewares.GetInstance(c)
 
-	// The sharing is refused if there is no access code or scope
-	sharingAccepted := scope != "" && accessCode != ""
+	// The sharing is refused if there is no access code
+	sharingAccepted := accessCode != ""
```

Comment:
I'm not sure if it is easy useful to use a local variable for `sharingAccepted`. The comment could be enough:

```go
// The sharing is refused if there is no access code
if accessCode != "" {
```

If you want to keep the local variable to describe the condition, you can do it like this:

```go
if sharingAccepted := accessCode != ""; sharingAccepted {
```